process_indices.ipynb

```diff
@@ -135,8 +149,19 @@
     "    #  pool-ing to then construct new Dataset\n",
     "    \n",
     "    with xr.open_dataset(fp) as ds:\n",
-    "        out = [indices.compute_index(ds[varname], index, model, scenario) for index in index_list]\n",
-    "    \n",
+    "        # out = [indices.compute_index(ds[varname], index, model, scenario) if index not in [\"wsdi\", \"csdi\"] else indices.compute_index(ds[varname], index, model, scenario) for index in index_list]\n",
```

Comment:
Can this commented out code be removed? If not, I'd suggest adding a comment above it to indicate why/when it should be uncommented.